main.go

```diff
@@ -1,48 +1,84 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"log"
 	"os"
 
 	"github.com/SUSE/scf-secret-generator/secrets"
 )
 
-func printHelp() {
-	fmt.Printf("Usage: %s <role-manifest>\n", os.Args[0])
-}
+var domain = flag.String(
+	"domain",
+	"",
+	"Domain",
+)
+
+var namespace = flag.String(
+	"namespace",
+	"",
+	"Kubernetes namespace",
+)
+
+var serviceDomainSuffix = flag.String(
+	"serviceDomainSuffix",
+	"",
+	"Service Domain Suffix",
+)
+
+var secretsName = flag.String(
+	"secretsName",
+	"",
+	"Secrets Name",
+)
+
+var secretsGeneration = flag.String(
+	"secretsGeneration",
+	"",
+	"Secrets Generation",
+)
+
+var certExpiration = flag.Int(
+	"certExpiration",
+	10950, // 30 years * 365 days
```

Comment:
Just say `30 * 365` (possibly with inline comments)?

Reply:
I added an additional 7 days to make sure it is slightly more than 30 years. Somehow I doubt anyone will ever run an instance for 30 years without ever rotating the certs...

> 640K should be enough for anyone!

Reply:
Ah, for the default value it's just to make the multiply explicit so it's easier to read.

The extra days thing is because actually specifying this in days could (separately) trip up people who specify this manually.